packages/cli/src/commands/link/__tests__/link-test.js

```diff
@@ -1,108 +1,93 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @format
- * @emails oncall+javascript_foundation
- */
+import {func as link} from '../link';
```

Comment:
I guess it's too much of a hassle to add Flow here?

Reply:
Yeah, it complains about "config" missing some properties. Don't want to define all properties for the sake of the test.